monitor/core/pom.xml

```diff
@@ -20,15 +20,181 @@
+      		<artifactId>brooklyn-rest-client</artifactId>
+    	</dependency> 
+    	<!--<dependency>
+    		<groupId>io.brooklyn</groupId>
+    		<artifactId>brooklyn-rest-client</artifactId>
+    		<version>0.7.0-M1</version>
+		</dependency>-->
+
+		 <dependency>
+			<groupId>com.googlecode.json-simple</groupId>
+			<artifactId>json-simple</artifactId>
+			<version>1.1</version>
+		</dependency>
+
+		<dependency>
+			<groupId>it.polimi.modaclouds.monitoring</groupId>
+			<artifactId>metrics-observer</artifactId>
+			<version>3.0.1</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>org.restlet</artifactId>
+					<groupId>org.restlet.jse</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>rsp-services-api</artifactId>
+					<groupId>polimi.deib</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>qos-models</artifactId>
+					<groupId>it.polimi.modaclouds.qos-models</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>knowledge-base-api</artifactId>
+					<groupId>it.polimi.modaclouds.monitoring.knowledge-base-api</groupId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<dependency>
+         	<groupId>monitoring-manager</groupId>
+         	<artifactId>monitoring-manager</artifactId>
+         	<scope>system</scope>
+         	<version>1.4</version>
+         	<systemPath>${basedir}/lib/monitoring-manager-1.4.jar</systemPath>
+      	</dependency>
+
+      	<dependency>
+      		<groupId>data-collector</groupId>
+			<artifactId>data-collector</artifactId>
+			<scope>system</scope>
+			<version>1.3-SNAPSHOT</version>
+			<systemPath>${basedir}/lib/data-collector-1.3-SNAPSHOT.jar</systemPath>
+		</dependency>
+	</dependencies>
+
+
+	<build>
+		<plugins>
+			<plugin>
+    			<artifactId>maven-antrun-plugin</artifactId>
+    			<executions>
+        			<execution>
+            			<id>download-files</id>
+            			<!--<phase>clean</phase>-->
+            			<phase>process-resources</phase>
+            			<goals>
+                			<goal>run</goal>
+            			</goals>
+            			<configuration>
+                			<target>
+                    			<!-- download files -->
+                    			<get src="http://archive.apache.org/dist/jena/binaries/jena-fuseki-1.1.1-distribution.zip"
+                         			dest="${project.build.directory}"
+                         			verbose="false"
+                         			usetimestamp="true"/>
+                    			<get src="https://magelan.googlecode.com/files/hyperic-sigar-1.6.4.zip"
+	        			            dest="${project.build.directory}"
+    	                		    verbose="false"
+        	                 		usetimestamp="true"/>
+                    			<get src="http://www.cs.uoi.gr/~dathanas/rsp-services-csparql-0.4.6.2-modaclouds-distribution.zip"
+                         			dest="${project.build.directory}"
+                         			verbose="false"
+                         			usetimestamp="true"/>
```

Comment:
These 3 ant artifacts apparently can be found in maven central repo:
- http://mvnrepository.com/artifact/org.apache.jena/jena-fuseki/1.1.1
- http://mvnrepository.com/artifact/org.fusesource/sigar/1.6.4
- http://streamreasoning.org/maven/polimi/deib/rsp-services-csparql/0.4.6/

I'm not sure about if the hyperic-sigar is the same artifact. Could you check that they are suitable for the project and replace them as Maven dependency? Thanks.